datasets/opus_paracrawl/README.md

```diff
@@ -0,0 +1,200 @@
+---
+annotations_creators:
+- found
+language_creators:
+- found
+languages:
+- bg
+- ca
+- cs
+- da
+- de
+- el
+- en
+- es
+- et
+- eu
+- fi
+- fr
+- ga
+- gl
+- ha
+- hr
+- hu
+- ig
+- is
+- it
+- km
+- lt
+- lv
+- mt
+- my
+- nb
+- ne
+- nl
+- nn
+- pl
+- pt
+- ro
+- ru
+- si
+- sk
+- sl
+- so
+- sv
+- sw
+- tl
+
+
+licenses:
+- unknown
+multilinguality:
+- multilingual
+size_categories:
+- 10K<n<100K
+source_datasets:
+- original
+task_categories:
+- conditional-text-generation
+task_ids:
+- machine-translation
+---
+
+# Dataset Card Creation Guide
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** http://opus.nlpl.eu/ParaCrawl.php
+- **Repository:** None
+- **Paper:** http://www.lrec-conf.org/proceedings/lrec2012/pdf/463_Paper.pdf
+- **Leaderboard:** [More Information Needed]
+- **Point of Contact:** [More Information Needed]
+
+### Dataset Summary
+
+
+To load a language pair which isn't part of the config, all you need to do is specify the language code as pairs.
+You can find the valid pairs in Homepage section of Dataset Description: http://opus.nlpl.eu/ParaCrawl.php
+E.g.
+
+`dataset = load_dataset("paracrawl", lang1="en", lang2="so")`
+
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+[More Information Needed]
+
+## Dataset Structure
+
+### Data Instances
+
+[More Information Needed]
+
+### Data Fields
+
+[More Information Needed]
```

Comment:
Would be nice to have a short description of the features here, you can generate templates using the latest version of `dataset-tagging` app